nannou/src/draw/renderer/mod.rs

```diff
@@ -82,7 +82,7 @@ pub struct Renderer {
     glyph_cache_texture: wgpu::Texture,
     depth_texture: wgpu::Texture,
     depth_texture_view: wgpu::TextureView,
-    default_texture: wgpu::Texture,
+    _default_texture: wgpu::Texture,
```

Comment:
probably should just be removed?

Reply:
Yeah I was thinking this, though I couldn't quite recall if we had to keep them in memory and if their behaviour changes if they're dropped earlier e.g. :thinking: I guess we wouldn't get these warnings if they had a Drop implementation? But I'm not 100% certain of that :sweat_smile: